### ISSUE TITLE
Google Old Homepage + Gmail´s normal mode doesn't seem to work

### ISSUE THREAD

**Issue by LeoKids** [rn10950/RetroZilla#55]
![VirtualBox_Windows NT 4 0_05_11_2021_12_21_50](https://user-images.githubusercontent.com/68135774/140535187-876f7748-99a4-4c17-854c-1c5a00cf79fc.png)

Altough Google worked in Retrozilla,it seemed that the google homepage is from the 2000s and when you search,it is in mobile.

Not sure if this is really true,but it seems that Google works with older browsers using the basic browser mode (transforming it into something of 2000s)

And when you search using the bulit-in search bar,it moves to google mobile version,did it happen since late 2010s?

![VirtualBox_Windows NT 4 0_05_11_2021_12_24_54](https://user-images.githubusercontent.com/68135774/140535739-bb35f4b8-c54e-4e1a-a471-b94ea19a90e6.png)

But...when it comes to the Gmail,the webgl isnt supported resulting to this.

![VirtualBox_Windows NT 4 0_05_11_2021_12_26_20](https://user-images.githubusercontent.com/68135774/140535905-75d2a299-51ae-4d95-9060-7c650c4c0fd6.png)

It did work with the basic mode,making Retrozilla (for now) stuck into the version for slow internets and legacy browsers.

![VirtualBox_Windows NT 4 0_05_11_2021_12_27_21](https://user-images.githubusercontent.com/68135774/140536099-f832ef75-0f02-46f5-baed-612ad72ed513.png)

Not sure if you guys will fix it?

**Comment by LeoKids** [comment]
It's been a while but without Basic mode,Gmail won't work,and it's now discontinued,so Gmail won't work anymore

**Comment by LeoKids** [comment]
Google Search is now broken,the mobile search seems to not work anymore...

**Comment by LeoKids** [comment]
@rn10950

**Comment by rn10950** [comment]
Yeah, gmail is out of the question. I would say use POP/IMAP but it seems they disabled that too....

Google tracking issue #85, closing this as duplicate

**Comment by LeoKids** [comment]
The tracking issue that you opened was actually a duplicate of this one but you still made this a duplicate of #85 cause of touching similar issues. This was back in 2021 and things must have changed since then...

**Comment by rn10950** [comment]
Yes, I'm closing all google-related issues and consolidating them on to a new one to keep all progress in one area.
